.github/workflows/build-test-debug.yml

```
@@ -50,11 +50,11 @@ jobs:
     - name: Run Content.Tests
       run: dotnet test --no-build --configuration DebugOpt Content.Tests/Content.Tests.csproj -- NUnit.ConsoleOut=0
 
-    # - name: Run Content.IntegrationTests
-    #   shell: pwsh
-    #   run: |
-    #     $env:DOTNET_gcServer=1
-    #     dotnet test --no-build --configuration DebugOpt Content.IntegrationTests/Content.IntegrationTests.csproj --filter FullyQualifiedName!~ShipyardTest -- NUnit.ConsoleOut=0 NUnit.MapWarningTo=Failed
+    - name: Run Content.IntegrationTests
+      shell: pwsh
+      run: |
+        $env:DOTNET_gcServer=1
+        dotnet test --no-build --configuration DebugOpt Content.IntegrationTests/Content.IntegrationTests.csproj --filter "FullyQualifiedName!~ShipyardTest&FullyQualifiedName!~TestWindows&FullyQualifiedName!~UiInteractTest&FullyQualifiedName!~SpawnAndDeleteEntityCountTest&FullyQualifiedName!~SpawnAndDirtyAllEntities" -- NUnit.ConsoleOut=0 NUnit.MapWarningTo=Failed
```

```suggestion
        dotnet test --no-build --configuration DebugOpt Content.IntegrationTests/Content.IntegrationTests.csproj --filter "FullyQualifiedName!~ShipyardTest" -- NUnit.ConsoleOut=0 NUnit.MapWarningTo=Failed
```
Will need to look up the individual tests, but this should do it.

---

Stinky